src/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -1096,6 +1096,31 @@ public override string ToString()
             }
         }
 
+        /// <summary>
+        ///   Get a JsonElement which can be safely stored beyond the lifetime of the
+        ///   original <see cref="JsonDocument"/>.
+        /// </summary>
+        /// <returns>
+        ///   A JsonElement which can be safely stored beyond the lifetime of the
+        ///   original <see cref="JsonDocument"/>.
+        /// </returns>
+        /// <remarks>
+        ///   If this JsonElement is itself the output of a previous call to Clone, or
+        ///   a value contained within another JsonElement which was the output of a previous
+        ///   call to Clone, this method results in no additional memory allocation.
```

Comment:
What about the ctor allocation - `new JsonDocument(...)`?

Reply:
If this JsonElement is the output of Clone then it just says `return this`.  There's no new document made.